product_supplierinfo_operating_unit/security/product_supplierinfo.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data noupdate="1">
+
+        <record id="ir_rule_purchase_order_allowed_operating_units" model="ir.rule">
+            <field name="model_id" ref="purchase.model_purchase_order" />
+            <field
+                name="domain_force"
+            >['|',('operating_unit_id','=',False),('operating_unit_id','in',[g.id for g in
+                user.operating_unit_ids])]
+            </field>
+            <field name="name">PO's from allowed operating units</field>
+            <field name="global" eval="True" />
+            <field eval="1" name="perm_unlink" />
+            <field eval="1" name="perm_write" />
+            <field eval="1" name="perm_read" />
+            <field eval="1" name="perm_create" />
+        </record>
```

Comment:
I don't think purchase order rules belong in this module. They are probably already in purchase_operating_unit of which this module should be a dependency. Pretty sure this came out of our custom purchase_operating_unit module.